initialization.tex

```diff
@@ -410,71 +410,70 @@ \subsection{Generating Lateral Boundary Data}
 \notag
 \end{equation}
 
-\noindent which would take a grid point from the initial value to the value at the next large-scale time 
-during 3 simulation hours.
+\noindent which would take a grid point from the initial value to the value at the 3-hour simulation time. 
 The horizontal momentum fields are coupled both with  $\mu_d$ and the inverse map factor.  The 
 other 3-dimensional fields ($\theta$, $q_v$, and $\phi'$) are coupled only with $\mu_d$.
 The $\mu'_d$ lateral boundary field is not coupled.
 
 Each lateral boundary field
 is defined along the four sides of the 
 rectangular grid (loosely referred to as the north, south, east, and west sides).  
-The boundary values and tendencies for vertical velocity and the non-vapor moisture species are included
+Boundary values and tendencies for vertical velocity and the non-vapor moisture species are included
 in the external lateral boundary file, but act as
 place-holders for the nested boundary data for the fine grids.
 The width of the lateral
-boundary along each of the four sides is user selectable at run-time.
+boundary along each of the four sides is user-selectable at run-time.
 
 \subsection{Masking of Surface Fields}
 
-Some of the meteorological and static fields are ``masked''.  A masked field is one in which
-the values are typically defined only over water (e.g., sea surface temperature) or defined
+Some of the meteorological and static fields are "masked".  A masked field is one in which
```

Comment:
@davegill 
In American English, punctuation should always go inside the quotes.
https://www.grammarbook.com/punctuation/quotes.asp (see rule 4).
So this should be "masked."